tests/jiva/clone/provision_test.go

```diff
@@ -128,29 +128,29 @@ var _ = Describe("[jiva] TEST JIVA CLONE CREATION", func() {
 			cloneObj, err = pvc.NewBuilder().
 				WithName(cloneName).
 				WithAnnotations(cloneAnnotations).
-				WithNamespace(nsName).
-				WithStorageClass(scName).
+				WithNamespace(namespaceObj.Name).
+				WithStorageClass("openebs-snapshot-promoter").
```

Comment:
Should this be the generated name

Reply:
For the clone pvc we use the `openebs-snapshot-promoter` storageclass installed by openebs and is common for all.

Reply:
@AmitKumarDas `openebs-snapshot-promoter` is install by maya installer , and its common for all the clone PVC provisioning..
@shubham14bajpai can you create a var in a suite file and use it here ?